R/reportFE.R

```diff
@@ -1344,10 +1368,16 @@ reportFE <- function(gdx, regionSubsetList = NULL,
   )
 
 
-  ### temporary (!) industry non-energy use reporting
+  #### Non-energy Use Reporting ----
+
+ ### temporary (!) industry non-energy use reporting
   # note: only for REMIND-EU SSP2
 
-  if ("DEU" %in% getRegions(vm_prodFe) & indu_mod == 'subsectors') {
+  if ("DEU" %in% getRegions(vm_prodFe) & indu_mod == 'subsectors' & is.null(vm_demFENonEnergySector)) {
+
+      # some initializations required for building library with dplyr operations below
+      encar <- data <- value <- value_subsectors <- SSP <- Value_NonEn <- encar <- region <- period <- NULL
```

Comment:
This is bad practice.  Use tidyselect syntax (i.e. `'encar'`) or the `.data` pronoun instead.

Reply:
arg, I think Felix had deleted that before, and now I had wrongly kept it when pulling. I just deleted it. thank you! maybe @fschreyer would like to have a look?

Reply:
Well, I guess it will given you an error when building, if you do not have it, right? But leave it for now if you did not introduce it. Probably the whole section can be deleted soon anyways (see below).